admin_guide/scheduling/priority_preemption.adoc

```diff
@@ -12,67 +12,66 @@ toc::[]
 
 == Overview
 
-Pods can have priority. Priority indicates the importance of a Pod relative to other Pods. If a Pod cannot be scheduled, the scheduler tries to preempt (evict) lower priority Pods to make scheduling of the pending Pod possible.
+Pod priority indicates the importance of a pod relative to other pods. If a pod cannot be scheduled, the scheduler preempts (evicts) lower-priority pods to make scheduling of the pending pod possible.
 
-Priority also affects scheduling order of Pods and out-of-resource eviction ordering on the Node.Pod priority and preemption .
+Pod priority also affects scheduling order of pods and out-of-resource eviction ordering on the node.. 
 
-== How to use priority and preemption
+== About Pod Priority
 
-To use priority and preemption, follow these steps:
+When pod priority is enabled, scheduler orders pending pods by their priority and a pending pod is placed ahead of other pending pods with lower priority in the scheduling queue. As a result, the higher priority pod may by scheduled sooner that pods with lower priority if its scheduling requirements are met. If such pod cannot be scheduled, scheduler will continue and tries to schedule other lower priority pods.
```

Comment:
s/pod priority/this feature because, preemption is a scheduler flag whereas pod priority is an field in pod spec and we need both to be enabled.